pymc/distributions/discrete.py

```diff
@@ -1004,6 +1004,10 @@ def logp(value, good, bad, n):
         value : numeric
             Value(s) for which log-probability is calculated. If the log probabilities for multiple
             values are desired the values must be provided in a numpy array or Aesara tensor
+        good : integer, array_like or TensorVariable
+            Number of successful individuals in the population. Alias for parameter :math:`k`.
+        bad : integer, array_like or TensorVariable
+            Number of unsuccessful individuals in the population. Alias for :math:`N-k`.
```

Comment:
Should we also use "tensor_like of integer" here @OriolAbril ?

Reply:
We are in the process of removing the distribution methods from the documentation and leaving only the `dist` one. The Distribution class is still being refactored, see #5308 for some more details (I don't really understand what is going on very well) and those methods might be removed completely, and because they should not be used themselves _already_ but called via `pm.logp()` or `pm.logcdf`

I think it's best to leave them be for now.